systemtest/src/main/java/io/strimzi/systemtest/templates/crd/KafkaTemplates.java

```diff
@@ -252,21 +252,45 @@ public static KafkaBuilder kafkaWithCruiseControl(String clusterName, int kafkaR
         return defaultKafka(kafka, clusterName, kafkaReplicas, zookeeperReplicas)
             .editSpec()
                 .editKafka()
-                    // faster cluster model generation tuning: reduce reporting and metadata refresh intervals
```

Comment:
Why did you remove the comments? From my POV it is useful to have some information why these configs are in place (due to faster model generation)

Reply:
That comment is misleading, as the cluster model generation is not faster with them. The new method has a name which makes it clear that it will provide faster model generation (thanks to a much more aggressive tuning).

Reply:
Makes sense to me, but it would be great to have a comment why we have these configs in place for future reference

Reply:
Ok, that's fine. Do you have anything in mind?